scripts/runSemgrep.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Check for install/updates at https://github.com/returntocorp/semgrep
+
+source scripts/requireDocker.sh
+
+benchmark_version=$(scripts/getBenchmarkVersion.sh)
+semgrep_version=$(docker run --rm returntocorp/semgrep --version)
+result_file="/src/results/Benchmark_$benchmark_version-semgrep-v$semgrep_version.json"
+
+docker run --rm -v "${PWD}:/src" returntocorp/semgrep --config p/security-audit -q --json -o "$result_file" . > /dev/null
```

Comment:
This script seems to work fine, and produce results in /results that are properly scored.